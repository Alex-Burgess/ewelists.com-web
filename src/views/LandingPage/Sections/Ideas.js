import React from "react";
import { makeStyles } from "@material-ui/core/styles";
import RecentArticles from "components/Articles/RecentArticles.js";

import styles from "assets/jss/material-kit-pro-react/views/landingPage/ideasStyle.js";
const useStyles = makeStyles(styles);

export default function LandingIdeas(props) {
  const classes = useStyles();

  const recentArticles = [
    'newborn-baby-essentials-list',
    'hospital-bag-checklist'
  ];

  return (
    <div className={classes.section}>
      <div className={classes.container}>
        <RecentArticles
          articles={
            recentArticles
          }
        />
      </div>
    </div>
  );
}
