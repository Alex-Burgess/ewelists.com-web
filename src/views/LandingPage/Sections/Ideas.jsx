/*!

=========================================================
* Material Kit PRO React - v1.7.0
=========================================================

* Product Page: https://www.creative-tim.com/product/material-kit-pro-react
* Copyright 2019 Creative Tim (https://www.creative-tim.com)

* Coded by Creative Tim

=========================================================

* The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.

*/
import React from "react";
// nodejs library to set properties for components
import PropTypes from "prop-types";
import withStyles from "@material-ui/core/styles/withStyles";
import RecentArticlesMain from "custom/GiftIdeas/RecentArticlesMain.jsx";

import ideasStyle from "assets/jss/material-kit-pro-react/views/landingPageSections/ideasStyle.jsx";

class LandingIdeas extends React.Component {
  render() {
    const { classes } = this.props;

    const recentArticles = [
      {title: "Travel Gear", url: "/listideas/travelgear", img: 'travelgear.jpg', img_position_left: true,
      description_short: "Great items to make travelling with your little ones no fuss!",
      beginning_content: "The compact stroller/buggy is one of the most useful items for travelling. They are handy for all sorts, such as effortlessly moving around the city hopping on and off public transport, taking with you through the airport, or just keeping in the car..."},
      {title: "Nursery List", url: "/listideas/nursery", img: 'nurserylist.jpg', img_position_left: false,
      description_short: "All the items you need for your babyâ€™s bedroom",
      beginning_content: "Have you been gushing over the possibilities for the cute little nursery or area you want to create for your baby. The nesting phase is such a special time in the lead up to the arrival of your baby and choosing how to decorate your nursery can bring immense joy..."}
    ];

    return (
      <div className={classes.section}>
        <div className={classes.container}>
          <RecentArticlesMain
            articles={
              recentArticles
            }
          />
        </div>
      </div>
    );
  }
}

LandingIdeas.propTypes = {
  classes: PropTypes.object
};

export default withStyles(ideasStyle)(LandingIdeas);
