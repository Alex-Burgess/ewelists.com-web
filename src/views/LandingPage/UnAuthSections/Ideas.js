import React from "react";
import { makeStyles } from "@material-ui/core/styles";
import RecentArticlesMain from "custom/GiftIdeas/RecentArticlesMain.js";

import styles from "assets/jss/custom/views/landingPage/ideasStyle.js";
const useStyles = makeStyles(styles);

export default function LandingIdeas(props) {
  const classes = useStyles();

  const recentArticles = [
    {title: "Christmas Ideas for Toddlers", url: "/listideas/christmasfortoddlers", img: 'christmastoddlers.jpg', img_position_left: true,
    description_short: "Great Christmas ideas for toddlers and young children",
    beginning_content: "If you are stuck wondering what to get a toddler for Christmas, here are some of our favourite ideas! Toddlers love to mimic adults and join in ‘helping’ with every day tasks. Here are some great ideas to encourage role play..."},
    {title: "Nursery List", url: "/listideas/nursery", img: 'nurserylist.jpg', img_position_left: false,
    description_short: "All the items you need for your baby’s bedroom",
    beginning_content: "Have you been gushing over the possibilities for the cute little nursery or area you want to create for your baby. The nesting phase is such a special time in the lead up to the arrival of your baby and choosing how to decorate your nursery can bring immense joy..."}
  ];

  return (
    <div className={classes.section}>
      <div className={classes.container}>
        <RecentArticlesMain
          articles={
            recentArticles
          }
        />
      </div>
    </div>
  );
}
