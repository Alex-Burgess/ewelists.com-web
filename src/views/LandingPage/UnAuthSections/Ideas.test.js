import * as React from 'react';
import TestRenderer from 'react-test-renderer';
import Ideas from './Ideas';

describe('Tests for Ideas section.', () => {
  test('Renders ideas section', () => {
    const tree = TestRenderer
      .create(<Ideas />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
