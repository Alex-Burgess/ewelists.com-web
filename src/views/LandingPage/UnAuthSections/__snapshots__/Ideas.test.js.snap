// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests for Ideas section. Renders ideas section 1`] = `
<div
  className="makeStyles-section-2"
>
  <div
    className="makeStyles-container-1"
  >
    <div
      className="makeStyles-section-5"
    >
      <div
        className="MuiGrid-root makeStyles-grid-12  MuiGrid-container"
      >
        <div
          className="MuiGrid-root makeStyles-grid-116 makeStyles-mlAuto-3 makeStyles-mrAuto-4 MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-10 MuiGrid-grid-md-10"
        >
          <div>
            <div
              className="makeStyles-card-117 makeStyles-cardPlain-118 makeStyles-cardBlog-120"
            >
              <div
                className="MuiGrid-root makeStyles-grid-12  MuiGrid-container"
              >
                <div
                  className="MuiGrid-root makeStyles-grid-116  MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-5 MuiGrid-grid-md-5"
                >
                  <div
                    className="makeStyles-cardHeader-132 makeStyles-cardHeaderPlain-133 makeStyles-cardHeaderImage-134"
                  >
                    <a
                      href="/listideas/christmasfortoddlers"
                    >
                      <img
                        alt="..."
                        className="makeStyles-leftImage-9"
                        src="christmastoddlers.jpg"
                      />
                    </a>
                    <div
                      style={
                        Object {
                          "backgroundImage": "url(christmastoddlers.jpg)",
                          "opacity": "1",
                        }
                      }
                    />
                  </div>
                </div>
                <div
                  className="MuiGrid-root makeStyles-grid-116  MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-7 MuiGrid-grid-md-7"
                >
                  <div
                    className="makeStyles-defaultFontStyle-144 makeStyles-infoText-151"
                  >
                    <h6
                      className="makeStyles-cardCategory-7"
                    >
                      <a
                        href="/listideas/christmasfortoddlers"
                      >
                        Christmas Ideas for Toddlers
                      </a>
                    </h6>
                  </div>
                  <h3
                    className="makeStyles-cardTitle-6"
                  >
                    <a
                      href="/listideas/christmasfortoddlers"
                    >
                      Great Christmas ideas for toddlers and young children
                    </a>
                  </h3>
                  <p
                    className="makeStyles-description1-8"
                  >
                    If you are stuck wondering what to get a toddler for Christmas, here are some of our favourite ideas! Toddlers love to mimic adults and join in ‘helping’ with every day tasks. Here are some great ideas to encourage role play...
                    <a
                      href="/listideas/christmasfortoddlers"
                    >
                       
                      Read More
                       
                    </a>
                  </p>
                </div>
                <div
                  className="MuiGrid-root makeStyles-grid-116  MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-5 MuiGrid-grid-md-5"
                >
                  <div
                    className="makeStyles-cardHeader-132 makeStyles-cardHeaderPlain-133 makeStyles-cardHeaderImage-134"
                  >
                    <a
                      href="/listideas/christmasfortoddlers"
                    >
                      <img
                        alt="..."
                        className="makeStyles-leftImageMobile-10"
                        src="christmastoddlers.jpg"
                      />
                    </a>
                    <div
                      style={
                        Object {
                          "backgroundImage": "url(christmastoddlers.jpg)",
                          "opacity": "1",
                        }
                      }
                    />
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div>
            <div
              className="makeStyles-card-117 makeStyles-cardPlain-118 makeStyles-cardBlog-120"
            >
              <div
                className="MuiGrid-root makeStyles-grid-12  MuiGrid-container"
              >
                <div
                  className="MuiGrid-root makeStyles-grid-116  MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-7 MuiGrid-grid-md-7"
                >
                  <div
                    className="makeStyles-defaultFontStyle-157 makeStyles-dangerText-167"
                  >
                    <h6
                      className="makeStyles-cardCategory-7"
                    >
                      <a
                        href="/listideas/nursery"
                      >
                        Nursery List
                      </a>
                    </h6>
                  </div>
                  <h3
                    className="makeStyles-cardTitle-6"
                  >
                    <a
                      href="/listideas/nursery"
                    >
                      All the items you need for your baby’s bedroom
                    </a>
                  </h3>
                  <p
                    className="makeStyles-description1-8"
                  >
                    Have you been gushing over the possibilities for the cute little nursery or area you want to create for your baby. The nesting phase is such a special time in the lead up to the arrival of your baby and choosing how to decorate your nursery can bring immense joy...
                    <a
                      href="/listideas/nursery"
                    >
                       
                      Read More
                       
                    </a>
                  </p>
                </div>
                <div
                  className="MuiGrid-root makeStyles-grid-116  MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-5 MuiGrid-grid-md-5"
                >
                  <div
                    className="makeStyles-cardHeader-132 makeStyles-cardHeaderPlain-133 makeStyles-cardHeaderImage-134"
                  >
                    <a
                      href="/listideas/nursery"
                    >
                      <img
                        alt="..."
                        className="makeStyles-rightImage-11"
                        src="nurserylist.jpg"
                      />
                    </a>
                    <div
                      style={
                        Object {
                          "backgroundImage": "url(nurserylist.jpg)",
                          "opacity": "1",
                        }
                      }
                    />
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
